### PR TITLE
Added logos

### PR DESCRIPTION
**Pull request by melloware**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+<img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/quarkus.svg" height="50" ><img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/plus-sign.svg" height="50" ><img src="https://github.com/quarkiverse/quarkus-quinoa/blob/main/docs/modules/ROOT/assets/images/node-logo.svg" height="50" >
```
Comment (shivam-sharma7) [inline README.md:10]: image does not appear in README.md

image
****

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]
Fix Path of images

**Comment by melloware** [comment]
Yeah I don't think it will work until the images are actually there and merged with this PR.

**Comment by melloware** [comment]
@shivam-sharma7 if you want to submit this same PR I can approve it.  I just can't approve and merge my own PR's.

**Comment by shivam-sharma7** [comment]
> @shivam-sharma7 if you want to submit this same PR I can approve it. I just can't approve and merge my own PR's.

no problem will wait till @ia3andy does not approves this PR.

**Review by ia3andy — APPROVED** [review]
Thanks

**Pull request merged**